src/olympia/addons/models.py

```diff
@@ -1129,7 +1129,9 @@ def can_request_review(self):
         latest_version = self.find_latest_version(
             amo.RELEASE_CHANNEL_LISTED, exclude=(amo.STATUS_BETA,))
 
-        return latest_version is not None and latest_version.files.exists()
+        return (latest_version is not None and
+                not latest_version.is_webextension and
+                latest_version.files.exists())
```

Comment:
Okay, didn't read farther up.  So, the issue conversation was misleading then - we were all talking about Public add-ons but they were already excluded, meaning this patch _only_ excludes webextensions.